Automattic/wp-calypso
# Add payment fees to plans grid

<img width="1326" alt="Screenshot 2023-06-14 at 11 53 03" src="https://github.com/Automattic/wp-calypso/assets/87168/f9a73cf4-2fde-4206-af91-74a2a1136444">

<img width="1314" alt="Screenshot 2023-06-14 at 11 53 29" src="https://github.com/Automattic/wp-calypso/assets/87168/f8c923d4-2275-46b3-b800-8401077815f5">


## Proposed Changes

* Add commission fees to plans grid for each plan, including free
* Add commission fees to comparison grid

## Testing Instructions



1. Check out this branch and run `yarn` and `yarn start` if needed. 
2. Navigate to the 2023 pricing grid in onboarding http://calypso.localhost:3000/start/plans
3. Verify that payment fees info has been added to the features grid and the comparison grid
4. Navigate to the 2023 pricing grid in the plans page http://calypso.localhost:3000/plans/{SITE_SLUG}
5. Repeat step 3

## Pre-merge Checklist
